next/backend/forms/test/schema.json

```diff
@@ -138,22 +138,21 @@
       "properties": {
         "chooseOneOf": {
           "title": "Choose one of radio buttons",
+          "type": "object",
+          "required": ["chooseOne"],
           "properties": {
             "chooseOne": {
               "type": "string",
-              "default": "multiply",
               "title": "Some random choose",
               "oneOf": [
                 {
                   "const": "screen",
                   "title": "Screen",
-                  "disabled": true,
-                  "tooltip": "This is some tooltip1"
+                  "tooltip": "Screen tooltip"
```

Comment:
based on today's discussions, let's move this into uiSchema - in the end, it's just a visual thing unrelated to data validationb, therefore it makes sense to clump it with other "visual stuff"